pycbc/distributions/joint.py

```diff
@@ -117,12 +119,17 @@ def __init__(self, variable_args, *distributions, **kwargs):
 
             # evaluate constraints
             result = numpy.ones(len(tuple(samples.values())[0]), dtype=bool)
-            for constraint in self._constraints:
-                result = constraint(samples) & result
+            for n in numpy.arange(n_test_samples):
+                sample = {param: samples[param][n] for param in distparams}
+                for constraint in self._constraints:
+                    result[n] = constraint(sample) & result[n]
 
             # set new scaling factor for prior to be
             # the fraction of acceptances in random sampling of entire space
             self._pdf_scale = sum(result) / float(n_test_samples)
+            if self._pdf_scale == 0.0:
+                raise ValueError("None of the random draws for pdf "
+                    "renormalization satisfied the constraints.")
```

Comment:
Thanks for catching this! Could you add a message to the error message telling the user to try to increase the number of test points by setting `n_test_samples` to something large than its current value (giving the current value)? That way they know what to do if they run into this error.